package.json

```diff
@@ -14,6 +14,7 @@
     "inquirer": "^3.3.0",
     "nodesecurity-npm-utils": "^6.0.0",
     "semver": "^5.4.1",
+    "strip-json-comments": "2.0.1",
```

Comment:
do we really need to pin to 2.0.1 specifically? `^2.0.1` would be more in line with what we would typically do